rplugin/python3/deoplete/child.py

```diff
@@ -484,6 +484,8 @@ def _set_source_attributes(self, context):
             'matchers',
             'max_abbr_width',
             'max_candidates',
+            'max_abbr_width',
+            'max_info_width',
```

Comment:
The code is needed.  But please sort the order.
`max_info_width` must be after `max_kind_width`.